core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -6435,10 +6435,38 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                     case SIMD_v128_store16_lane:
                     case SIMD_v128_store32_lane:
                     case SIMD_v128_store64_lane:
+                    {
+                        wasm_set_exception(module, "unsupported SIMD opcode");
+                        break;
+                    }
+#define SIMD_LOAD_ZERO_OP(width)                   \
+    do {                                           \
+        uint32 offset, addr;                       \
+        offset = read_uint32(frame_ip);            \
+        int32 base = POP_I32();                    \
+        offset += base;                            \
+        addr = GET_OPERAND(uint32, I32, 0);        \
+        addr_ret = GET_OFFSET();                   \
+        CHECK_MEMORY_OVERFLOW(width / 8);          \
+        V128 v;                                    \
+        memset(&v, 0, sizeof(v));                  \
```

Comment:
can probably be optimized to `memset(&v, width/8, sizeof(v) - width/8)`

Reply:
this seems to be more concise:
```suggestion
        V128 v = {0};
```

Reply:
This is probably more concise but doesn't allow to avoid setting first 4/8 bytes which means it does up to 50% more work to set the memory,
So I propose to go with memset still 

Let me know what you think

Reply:
Not sure I understand your concerns. The entire structure will be initialized to 0, see https://stackoverflow.com/questions/11152160/initializing-a-struct-to-0

Reply:
Yeah but I don't need first 4/8 bytes to be 0 initialised. I am going to overwrite them later.

In that case I don't want to zero them first for no reason and I only want to memset to 0 the part which will stay (remaining 12 or 8 bytes respectively)

Reply:
The existing code resets all the bytes, doesn't skip any of that. Also, the improvement you propose looks to me more like a micro pre-mature optimization; unless you have any solid benchmark results that prove the difference, I don't think the code complexity is worth it.